deployer/src/handlers/local.rs

```diff
@@ -32,17 +32,9 @@ pub async fn set_jwt_bearer<B>(
     mut request: Request<B>,
     next: Next<B>,
 ) -> Result<Response, StatusCode> {
-    let mut auth_details = None;
-
     if let Some(bearer) = request.headers().typed_get::<Authorization<Bearer>>() {
```

Comment:
Should we update the error in the `else` clause to reflect that a cookie is not an option anymore?

Reply:
Yip, I updated the error and remove all other references to "cookie" :smile: